AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -429,3 +433,27 @@ extension GiftCardComponent: PartialPaymentComponent {}
 
 @_spi(AdyenInternal)
 extension GiftCardComponent: PublicKeyConsumer {}
+
+extension GiftCardComponent: SubmitCustomizable {
+
+    // MARK: - SubmitCustomizable
+
+    /**
```

Comment:
Could be removed because the protocol propagates it

Reply:
Done